models/board.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+from peewee import Model, CharField, IntegrityError, DateTimeField
+
+from . import DB
+
+
+# Pin을 모아두는 Board 모델 정의
+class Board(Model):
+    title = CharField(max_length=20, primary_key=True)
+    comment = CharField(max_length=200)
+    created_at = DateTimeField(default=datetime.datetime.now())
+
+    class Meta:
+        database = DB
+
+    # C create board
+    @classmethod
+    def create_board(cls, title, comment):
+        try:
+            board = cls.create(title=title, comment=comment)
```

Comment:
정의만 되고 사용하지 않고 있는데, 필요 없다면 제거하는 게 나을 것 같습니다.
```suggestion
cls.create(title=title, comment=comment)
```